base/math.jl

```diff
@@ -465,6 +479,8 @@ log(x::Number)
 Compute the logarithm of `x` to base 2. Throws [`DomainError`](@ref) for negative
 [`Real`](@ref) arguments.
 
+See also: [`exp2`](@ref), [`ldexp`](@ref).
+
```

Comment:
Note, in the merge, I lost the following corresponding update to `exp2`, when that doc moved from here to special/exp.jl:

```md
See also [`ldexp`](@ref), [`<<`](@ref).
```

Reply:
bump

Reply:
Now restored, sorry! Also added link from `exp(x)`